src/lib/storage.js

```diff
@@ -9,7 +10,8 @@ const fromStorage = (store) => {
   return storage.getItem('username')
     .then((username) => store.dispatch(loginSetUsername(username)))
     .then(() => storage.getItem('password'))
-    .then((password) => store.dispatch(loginSetPassword(password)));
+    .then((password) => store.dispatch(loginSetPassword(password)))
+    .then(refreshEvents);
```

Comment:
`refreshEvents()` definitely has to happen _after_ fixing up the username and password. That said, those 2 could be done in any order. I did consider a `Promise.all()` but I though the code would look too weird. I think I'll leave this for now unless you feel strongly about it.